packages/rest/src/keys.ts

```diff
@@ -62,6 +63,13 @@ export namespace RestBindings {
    * Internal binding key for http-handler
    */
   export const HANDLER = BindingKey.create<HttpHandler>('rest.handler');
+  /**
+   * Binding key for setting and injecting Reject action's error handling
+   * options
+   */
+  export const ERROR_HANDLER_OPTIONS = BindingKey.create<ErrorHandlerOptions>(
+    'rest.sequence.actions.reject.options',
```

Comment:
Can we remove a reference to sequence actions from the key name too? E.g. `rest.errorHandlerOptions`.